app/default/props.conf

```diff
@@ -19,24 +19,37 @@ TRANSFORMS-source_type = heroku_system_source_type, heroku_requests_source_type,
 # Do not index some high volume low value messages.
 TRANSFORMS-heroku_requests_null_queue = heroku_requests_null_queue
 
-[heroku_system]
+[heroku:system]
 category = Structured
 description = Messages about actions taken by the Heroku platform.
 
-EXTRACT-heroku_component,dyno = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
+EXTRACT-component,dyno = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
 EXTRACT-message = \s-\s(?<message>.*)
 EXTRACT-state = State changed from \w+ to (?<state>\w+)$
 EXTRACT-exit_code = Process exited with status (?<exit_code>\w+)$
 
-[heroku_requests]
+[heroku:router]
 category = Structured
 description = Request logs produced by the Heroku HTTP routers.
 
 # Note: Only extract component as request logs contain a dyno field with a value that aligns with the other Heroku source types. Would always be "router" otherwise.
-EXTRACT-heroku_component = ^\S+\s(?<component>\S+)
-
-[heroku_api]
+EXTRACT-component = ^\S+\s(?<component>\S+)
+
+# Splunk CIM fields for the web model, see https://docs.splunk.com/Documentation/CIM/5.0.1/User/Web and https://devcenter.heroku.com/articles/http-routing#info-logs.
+# The `status` field already exists in the log message.
+FIELDALIAS-http_method = method AS http_method
+FIELDALIAS-src = fwd AS src
+FIELDALIAS-dest = host AS dest
+FIELDALIAS-bytes_out = bytes AS bytes_out
+FIELDALIAS-uri_path = path AS uri_path
+EXTRACT-duration = service=(?<duration>\d+)ms
+EVAL-url = protocol . "://" . host . path
+
+[heroku:api]
 category = Structured
 description = Messages about Heroku app administrative actions taken by users.
 
-EXTRACT-heroku_component,dyno = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
+EXTRACT-component,dyno = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
+
+# Splunk CIM fields for the change model, see https://docs.splunk.com/Documentation/CIM/5.0.1/User/Change.
+EXTRACT-user = by user (?<user>\S+)
```

Comment:
An initial field extraction for the Change model.

There is lots of good stuff in this one so I'll raise an issue to capture what we could do with these messages.